### PR TITLE
Improve findings for Mirai, vncjew, alfa, custom RAT

### PR DESCRIPTION
**Pull request by tstromberg**
Also addresses more false-positives on Ubuntu discovered during rule development. 

Related: https://github.com/chainguard-dev/malcontent-samples/pull/20

### CODE REVIEW

File: rules/ref/words/keylogger.yara [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+rule keylogger : medium {
+  meta:
+    description = "references a 'keylogger'"
+  strings:
+      $ref = /[\w\_]{0,64}[kK]eylogger[\w\_]{0,64}/ fullword
```
Comment (egibs) [inline rules/ref/words/keylogger.yara:6]: Nit: the spacing here and on L15 looks a bit off.

**Comment by egibs** [comment]
FYI -- the tests are failing with:
```
--- FAIL: TestSimple/linux/2024.gas/gas (0.00s)
        samples_test.go:122: test program missing: linux/2024.gas/gas
            contents of linux/2024.gas: 
```

**Comment by tstromberg** [comment]
> FYI -- the tests are failing with:
> 
> ```
> --- FAIL: TestSimple/linux/2024.gas/gas (0.00s)
>         samples_test.go:122: test program missing: linux/2024.gas/gas
>             contents of linux/2024.gas: 
> ```

it needs a samples update - will update PR now that the samples have been merged.

**Comment by tstromberg** [comment]
@egibs - PTAL

**Review by egibs — APPROVED** [review]

**Pull request merged**